### DISCUSSION TITLE
FileNotFoundError: File not found: "data" pytorch_lightning trainer

### DISCUSSION THREAD

**Discussion by Muhammad-Raza156** [TorchIO-project/torchio#1083] [Q&A]
I am implementing the Task 05 of decathalon dataset which is prostate images by following the tutorial.
https://github.com/Project-MONAI/tutorials/blob/main/modules/TorchIO_MONAI_PyTorch_Lightning.ipynb

it is giving the error  "FileNotFoundError: File not found: "data"" 
 
start = datetime.now()
print("Training started at", start)
trainer.fit(model=model, datamodule=data)
print("Training duration:", datetime.now() - start)

**Comment by fepegar** [comment]
Hi, @Muhammad-Raza156. Can you please try the notebook in TorchIO?

https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_MONAI_PyTorch_Lightning.ipynb

**Reply by Muhammad-Raza156** [reply]
Thanks for the response it is working on Hippo data but i want to use it with Task05 Prostate medical decathlon dataset i have data in my google drive

**Comment by fepegar** [comment]
Have you checked that the dataset was downloaded correctly?

**Reply by Muhammad-Raza156** [reply]
Sorry for the late response i was busy with some more important stuff , yes i have downloaded data correctly because i am already  using in another colab notebook

**Reply by fepegar** [reply]
Can you share a minimal working example I can reproduce?
